pyproject.toml

```diff
@@ -0,0 +1,30 @@
+[tool.poetry]
+name = "d-fast-bank-erosion"
+version = "2.3.1"
+description = "A tool to perform a bank erosion analysis based on a number of D-Flow FM simulations."
+authors = ["Stichting Deltares <delft3d.support@deltares.nl>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
```

Comment:
Is het een idee om dit te sorteren en gelijk te trekken tussen BE/MI, zodat de verschillende packages onderop staan.
Of om dit in tweeën te delen met een specifieke [tool.poetry.dependencies] voor BE en MI.
E.g. Generic D-Fast & Specific BE/MI.